net-misc/apt-cacher-ng/apt-cacher-ng-9999999.ebuild

```diff
@@ -1,121 +1,95 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
-inherit cmake git-r3 user
+
+inherit cmake git-r3
 
 DESCRIPTION="Yet another caching HTTP proxy for Debian/Ubuntu software packages"
-HOMEPAGE="
-	https://www.unix-ag.uni-kl.de/~bloch/acng/
-	https://packages.qa.debian.org/a/apt-cacher-ng.html
-"
-LICENSE="BSD-4 ZLIB public-domain"
-SLOT="0"
+HOMEPAGE="https://www.unix-ag.uni-kl.de/~bloch/acng/
+	https://packages.qa.debian.org/a/apt-cacher-ng.html"
 EGIT_REPO_URI="https://salsa.debian.org/blade/apt-cacher-ng.git"
 EGIT_BRANCH="upstream/sid"
 
-KEYWORDS=""
-IUSE="doc fuse systemd tcpd"
+LICENSE="BSD-4 ZLIB public-domain"
+SLOT="0"
+IUSE="doc fuse tcpd"
 
-COMMON_DEPEND="
+DEPEND="acct-user/apt-cacher-ng
+	acct-group/apt-cacher-ng
 	app-arch/bzip2
-	app-arch/xz-utils
 	dev-libs/libevent:=
 	dev-libs/openssl:0=
 	sys-libs/zlib
-	systemd? (
-		sys-apps/systemd
-	)
-"
-BDEPEND="
-	${COMMON_DEPEND}
-	virtual/pkgconfig
-"
-RDEPEND="
-	${COMMON_DEPEND}
-	dev-lang/perl
-	fuse? ( sys-fs/fuse )
-	tcpd? ( sys-apps/tcp-wrappers )
-"
+	fuse? ( sys-fs/fuse:0 )
+	tcpd? ( sys-apps/tcp-wrappers )"
+BDEPEND="virtual/pkgconfig"
+RDEPEND="${DEPEND}
+	dev-lang/perl"
+
 PATCHES=(
-	"${FILESDIR}"/${PN}-3.5-perl-syntax.patch
-	"${FILESDIR}"/${PN}-9999999-flags.patch
+	"${FILESDIR}/${PN}-3.3.1-flags.patch"
+	"${FILESDIR}/${PN}-3.5-perl-syntax.patch"
 )
-S=${WORKDIR}/${P/_*}
 
-pkg_setup() {
-	# add new user & group for daemon
-	enewgroup ${PN}
-	enewuser ${PN} -1 -1 -1 ${PN}
+S="${WORKDIR}/${P/_*}"
+
+src_prepare() {
+	# Fixup systemd/CMakeLists.txt cmake version requirement
+	sed -ie "s/2.6/3.1/" systemd/CMakeLists.txt || die
+
+	# Make sure we install everything the same way it used to be after
+	# switching from mostly custom src_install to relying on build system
+	# installation
+	sed -ie "/install/s/LIBDIR/CFGDIR/" conf/CMakeLists.txt || die
+	sed -ie '/install.*acng\.conf/s/)$/ RENAME '"${PN}"'.conf)/' conf/CMakeLists.txt || die
+	sed -ie '/file/s/)$/ "*hooks" "backends_debian")/' conf/CMakeLists.txt || die
```

Comment:
Here too, preferably, although I could understand it in `-9999` being 3 different calls (for debug). Should it be renamed `-9999` while at it?